src/screens/DetailsScreen.tsx

```diff
@@ -81,49 +60,112 @@ export default function DetailsScreen({ route, navigation }: Props) {
                 ]
               );
             }}
+            icon="delete"
           >
-            Remove todo
+            Delete
           </Button>
         </View>
-        <View style={styles.containerItemColumn}>
-          <Text variant="bodyMedium" style={styles.boldText}>
-            Description
-          </Text>
-          <Text variant="bodyLarge">{todoData.description} </Text>
-        </View>
-        <View style={styles.containerItemColumn}>
-          <Text variant="bodyMedium" style={styles.boldText}>
-            Dates and times
-          </Text>
-          <Text variant="bodyLarge">
-            Due date: {HourMinutesFormater(todoData.dueDate)}{" "}
-            {todoData.dueDate.toDateString()}
-          </Text>
-          {todoData.alertTime ? (
-            <Text variant="bodyLarge">
-              alert date: {HourMinutesFormater(todoData.alertTime)}{" "}
-              {todoData.alertTime.toDateString()}
+        <View style={styles.container}>
+          <View style={styles.containerItemColumnAlignCenter}>
+            {todoData.imageUri ? (
+              <Image
+                source={{ uri: todoData.imageUri }}
+                style={{
+                  width: 200,
+                  height: 200,
+                  borderRadius: 999,
+                }}
+              />
+            ) : (
+              <View />
+            )}
+            <Text
+              variant="headlineLarge"
+              style={{
+                margin: 10,
+              }}
+            >
+              {todoData.title}
             </Text>
-          ) : (
-            <Text variant="bodyLarge">No alert time set.</Text>
-          )}
-        </View>
-        {todoData.location ? (
+          </View>
+          <View style={styles.containerItemRow}>
+            <Button
+              mode="contained"
+              icon={todoData.status === "Completed" ? "check" : "alert-circle"}
+              buttonColor={todoData.status === "Completed" ? "green" : "red"}
+              onPress={() => {
+                todoData.status =
+                  todoData.status === "Completed" ? "Pending" : "Completed";
+                updateTodo(todoData);
+              }}
+              style={{ width: "48%" }}
+            >
+              Status: {todoData.status}
```

Comment:
```suggestion
             {todoData.status}
```
så kan man se vad som står i knappen när den är complete